x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -229,7 +237,20 @@ export class TaskManagerPlugin
     executionContext,
     docLinks,
   }: CoreStart): TaskManagerStartContract {
-    const savedObjectsRepository = savedObjects.createInternalRepository(['task']);
+    const savedObjectsRepository = savedObjects.createInternalRepository([
+      TASK_SO_NAME,
+      BACKGROUND_TASK_NODE_SO_NAME,
+    ]);
+
+    this.kibanaDiscoveryService = new KibanaDiscoveryService({
+      savedObjectsRepository,
+      logger: this.logger,
+      currentNode: this.taskManagerId!,
+    });
+
+    if (this.shouldRunBackgroundTasks) {
+      this.kibanaDiscoveryService.start().catch(() => {});
```

Comment:
NIT: I see a lot of `.catch(() => {})` without even debug/trace level messages. We usually like debug/trace messages in such catch blocks.

Reply:
This because there is no scenario to fall in catch here, there is a try-catch in start method that prevents it to throw.
This is just to satisfy the linter.